packages/babel-plugin-proposal-private-property-in-object/src/index.js

```diff
@@ -6,15 +6,21 @@ import {
   FEATURES,
 } from "@babel/helper-create-class-features-plugin";
 
+import pluginPrivateIn from "./native-private-fields";
+
 export default declare((api, options) => {
   api.assertVersion(7);
 
+  const { loose, nativePrivateFields } = options;
+
+  if (nativePrivateFields) return pluginPrivateIn(api);
```

Comment:
can we default `nativePrivateFields` to `!isRequire("proposal-private-methods", api.targets())`?

Reply:
This isn't necessarily intended to be a public api right since we can just check targets? Or did we want to be able to specify that

Reply:
Ideally every plugin should automatically compile as few as possible based on the targets, but users should still be able to override it via options.